config/filter.d/openhab.conf

```diff
@@ -0,0 +1,15 @@
+# Openhab brute force auth filter: /etc/fail2ban/filter.d/openhab.conf:
+#
+# Block IPs trying to auth wp openhab
+#
+# Matches e.g.
+# 12.34.33.22 -  -  [26/sept./2015:18:04:43 +0200] "GET /openhab.app HTTP/1.1" 401 1382 
+#
+[Definition] 
+failregex = ^<HOST> -  - \s*\[\]\s* "GET /openhab.app HTTP/1.1" 401 1382.*$
```

Comment:
It seems that it should be possible to simply merge both regular expressions (path1|path2) and this avoid duplication etc